exporter/exporterparser/zipkin_test.go

```diff
@@ -112,6 +112,15 @@ exporters:
     {"timestamp": 1472470996403000,"value": "bar"}
   ],
   "tags": {"http.path": "/api","clnt/finagle.version": "6.45.0"}
+},
```

Comment:
side note: in principle this represents a slightly increase in the probability of the sporadic test failure that we saw in  #181